# pr/tests/sel_spd.c:427:20: error: implicit declaration of function 'getopt' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- nspr/pr/tests/sel_spd.c
+++ nspr/pr/tests/sel_spd.c
@@ -15,6 +15,9 @@
 #include <stdio.h>
 #include <errno.h>
 #include <string.h>
+
+extern char *optarg;
+int getopt(int argc, char *const argv[], const char *optstring);
 
 #ifdef DEBUG
 #define PORT_INC_DO +100
--- nspr/pr/tests/testfile.c
+++ nspr/pr/tests/testfile.c
@@ -23,6 +23,9 @@
 #include <getopt.h>
 #include <errno.h>
 #endif /* XP_OS2 */
+
+extern char *optarg;
+int getopt(int argc, char *const argv[], const char *optstring);
 
 static int _debug_on = 0;

